compiler_gym/wrappers/time_limit.py

```diff
@@ -32,12 +32,23 @@ def __init__(self, env: CompilerEnv, max_episode_steps: Optional[int] = None):
         self._max_episode_steps = max_episode_steps
         self._elapsed_steps = None
 
-    def step(self, action: ActionType, **kwargs):
+    # def step(self, action: ActionType, **kwargs):
+    #     assert (
+    #         self._elapsed_steps is not None
+    #     ), "Cannot call env.step() before calling reset()"
+    #     observation, reward, done, info = self.env.step(action, **kwargs)
+    #     self._elapsed_steps += 1
+    #     if self._elapsed_steps >= self._max_episode_steps:
+    #         info["TimeLimit.truncated"] = not done
+    #         done = True
+    #     return observation, reward, done, info
+
+    def multistep(self, actions: Iterable[ActionType], **kwargs):
```

Comment:
```suggestion
    def multistep(self, actions: Iterable[ActionType], **kwargs):
        actions = list(actions)
```

I would suggest forcing the conversion of actions to a list here, since there could be a subtle bug if actions is an iterator, as it would break the `len(actions)` below.